pydevices/HtsDevices/cryocon18i.py

```diff
@@ -82,12 +77,12 @@ def debugging(self):
 
 class CRYOCON18I_TREND(CRYOCON18I):
 
```

Comment:
Timo, I think we should leave this section unchanged. I remember that if we didn't do a shallow copy of the CRYOCON18I.parts it wouldn't work correctly. What will be the main reason for this purposed change?
Thanks.

Reply:
the + [ does the copy for you

Reply:
extra use of copy package .. unnecessary. list(parts would accomplish the same) and in this case using list + list is better for allocation